data/strings/sound.txt

```diff
@@ -409,12 +482,12 @@
     hu = Hajtson ki!
     id = Keluar.
     it = Uscita.
-    ja = 出口.
+    ja = 出口です
     ko = 출구.
     mr = बाहेर पडा.
     nb = Ta avkjøringen.
     nl = Verlaat.
-    pl = Zjazd.
+    pl = Wybierz zjazd
```

Comment:
@Acrobot line 490 now says `    pl = Zjedź zjazdem.` per your upstream changes, when this PR is merged it will only be used for the definitive stand-alone command like "in 100 meters, exit." Line 570 above for [take_exit_number] will be used in the fuller case of "in 100 meters, take exit 123, Main Street, Queen's Expressway, London" as opposed to "in 100 meters, turn right onto Main Street, Queen's Expressway, London"